torch/quantization/fx/prepare.py

```diff
@@ -166,6 +168,40 @@ def update_qconfig_for_qat(
             object_type_dict[all_qat_mappings[k]] = v
     return qconfig_dict
 
+def update_qconfig_for_fusion(
```

Comment:
Should we update the qconfig in the `fuse_fx` function?

Reply:
It's a little easier to update in the current location because at that point the qconfig_dict and object_type has been converted to an ordered dictionary. But we could also update in the fuse_fx function, but I think the logic will be a little more complicated because we cannot just use `object_type_dict.get`.